# ResultMsg.msg
int8 order
float64 reward
float64[] actions